code/modules/surgery/healing.dm

```diff
@@ -53,7 +53,9 @@
 /datum/surgery_step/heal/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	var/umsg = "You succeed in fixing some of [target]'s wounds"
 	var/tmsg = "[user] fixes some of [target]'s wounds"
-	if(get_location_accessible(target, target_zone))
+	if(target.stat == DEAD)
+		target.heal_bodypart_damage(brutehealing*1.5,burnhealing*1.5)
+  if(get_location_accessible(target, target_zone))
```

Comment:
fix your spacing

Reply:
Need to actually rewrite some of the code so that the death and clothed checks aren’t separate from one another, ill fix the